AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -179,14 +179,15 @@ public final class UPIComponent: PaymentComponent,
     }()
 
     /// The QRCode generation message item.
-    internal lazy var qrCodeGenerationLabelContainerItem: FormContainerItem = {
+    internal lazy var qrCodeGenerationLabelContainerItem: FormItem = {
         let item = FormLabelItem(text: localizedString(.UPIQrcodeGenerationMessage,
                                                        configuration.localizationParameters),
                                  style: configuration.style.footnoteLabel)
         item.style.textAlignment = .center
         item.identifier = ViewIdentifierBuilder.build(scopeInstance: self,
                                                       postfix: ViewIdentifier.generateQRCodeContainerItem)
-        return FormContainerItem(content: item.addingDefaultMargins())
+        
+        return item.padding().padding()
```

Comment:
I believe there is a good reason for double padding but from the code perspective just looks weird. Could this break in the future if current `padding()` implementation changes?

Reply:
it would change the look slightly but the behavior of padding is defined well enough to not accidentally cause this issue